custom_components/luxtronik/number_entities_predefined.py

```diff
@@ -463,15 +463,88 @@
     # endregion Solar
     # endregion Domestic water
     # region Cooling
-    # LuxtronikNumberDescription(
-    #     device_key=DeviceKey.cooling,
-    #     luxtronik_key=LP.P0108_MODE_COOLING,
-    #     key="cooling",
-    #     icon="mdi:snowflake",
-    #     entity_category=EntityCategory.CONFIG,
-    #     device_class=None,
-    #     on_state=LuxMode.automatic.value,
-    #     off_state=LuxMode.off.value,
-    # ),
+    LuxtronikNumberDescription(
+        key=SensorKey.COOLING_OUTDOOR_TEMP_THRESHOLD,
+        luxtronik_key=LP.P0110_COOLING_OUTDOOR_TEMP_THRESHOLD,
+        device_key=DeviceKey.cooling,
+        icon='mdi:sun-thermometer',
+        device_class=SensorDeviceClass.TEMPERATURE,
+        native_unit_of_measurement=UnitOfTemperature.CELSIUS,
+        native_min_value=18.0,
+        native_max_value=30.0,
+        native_step=0.5,
+        #entity_category=EntityCategory.CONFIG,
```

Comment:
Why not in config?

Reply:
I'm not sure what your suggesting here, can you explain?